2.0.0/amd64/debian/Dockerfile

```diff
@@ -112,3 +112,120 @@ ENTRYPOINT ["/entrypoint.sh"]
+#               NOTE: THIS DOCKERFILE IS GENERATED VIA "update.sh"
+#
+#                       PLEASE DO NOT EDIT IT DIRECTLY.
+# ------------------------------------------------------------------------------
+#
+FROM multiarch/$base_image
+
+# Set download urls
+ENV \
+    JAVA_URL="$java_url" \
+    OPENHAB_URL="$openhab_url" \
+    OPENHAB_VERSION="$version"
+
+# Set variables and locales
+ENV \
+    APPDIR="/openhab" \
+    EXTRA_JAVA_OPTS="" \
+    OPENHAB_HTTP_PORT="8080" \
+    OPENHAB_HTTPS_PORT="8443" \
+    LC_ALL="en_US.UTF-8" \
+    LANG="en_US.UTF-8" \
+    LANGUAGE="en_US.UTF-8" \
+    CRYPTO_POLICY="limited"
+
+# Set arguments on build
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+
+# Basic build-time metadata as defined at http://label-schema.org
+LABEL org.label-schema.build-date=$BUILD_DATE \
+    org.label-schema.docker.dockerfile="/Dockerfile" \
+    org.label-schema.license="EPL" \
+    org.label-schema.name="openHAB" \
+    org.label-schema.vendor="openHAB Foundation e.V." \
+    org.label-schema.version=$VERSION \
+    org.label-schema.description="An open source, technology agnostic home automation platform" \
+    org.label-schema.url="http://www.openhab.com/" \
+    org.label-schema.vcs-ref=$VCS_REF \
+    org.label-schema.vcs-type="Git" \
+    org.label-schema.vcs-url="https://github.com/openhab/openhab-docker.git" \
+    maintainer="openHAB <info@openhabfoundation.org>"
+
+# Install basepackages
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+    ca-certificates \
+    dirmngr \
+    fontconfig \
+    gnupg \
+    libpcap-dev \
+    locales \
+    locales-all \
+    netbase \
+    unzip \
+    wget \
+    zip && \
+    ln -s -f /bin/true /usr/bin/chfn
+
+# Install java
+ENV JAVA_HOME='/usr/lib/java-8'
+RUN wget -nv -O /tmp/java.tar.gz ${JAVA_URL} && \
+    mkdir ${JAVA_HOME} && \
+    tar --exclude='demo' --exclude='sample' --exclude='src.zip' -xvf /tmp/java.tar.gz --strip-components=1 -C ${JAVA_HOME} && \
+    rm /tmp/java.tar.gz && \
+    update-alternatives --install /usr/bin/java java ${JAVA_HOME}/bin/java 50 && \
+    update-alternatives --install /usr/bin/javac javac ${JAVA_HOME}/bin/javac 50
+
+# Install gosu
+ENV GOSU_VERSION 1.10
+RUN set -x \
+    && dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')" \
+    && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch" \
+    && wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc" \
+    && export GNUPGHOME \
+    && GNUPGHOME="$(mktemp -d)" \
+    && GPG_KEY="B42F6819007F00F88E364FD4036A9C25BF357DD4" \
+    && gpg --keyserver ha.pool.sks-keyservers.net --recv-keys $GPG_KEY \
+       || gpg --keyserver pgp.mit.edu --recv-keys $GPG_KEY \
+       || gpg --keyserver keyserver.pgp.com --recv-keys $GPG_KEY \
+    && gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu \
```

Comment:
All this gosu code was removed with https://github.com/openhab/openhab-docker/pull/180

Reply:
Perhaps it's better to create a branch from the master and apply your changes there?

Reply:
OK, up update from upstream/master

Reply:
Thanks. This looks a lot better! :smile: